src/server/FileParser/ProcessFiles.ts

```diff
@@ -129,112 +131,118 @@ function processFile(sourceFile: SourceFile) {
   return getCodeInformation(sourceFile);
 }
 
-function processClassDeclaration(node: Node) {
+interface IProcessedClassDeclaration {
+  properties: Properties;
+  methods: Methods;
+  name?: string;
+  modifiers?: string[];
+}
+
+function processClassDeclaration(classDeclaration: ClassDeclaration): IProcessedClassDeclaration {
```

Comment:
```suggestion
function processClassDeclaration(classDeclarationNode: ClassDeclaration): IProcessedClassDeclaration {
```

Think I want the extra `node` mentioned in the var here